operantanalysis/operantanalysis.py

```diff
@@ -7,15 +8,14 @@ def accessfiles(filename):
     :param filename: string that refers to operant file location, file is txt
     :return: subject number, list of time codes, and list of eventcodes
```

Comment:
and what will happen if later you discover that you need one (or more) of other fields from that file?

also I would recommend to name function like "load_file" not just "access...". moreover you are dealing with a single file at a time so make it single

Reply:
Resolved in most recent branch